### ISSUE TITLE
Add another mp campaign to mainline.

### ISSUE THREAD

**Issue by gfgtdf** [wesnoth/wesnoth#1953]
It'd be nice if we would have another mp campaign in mainline, LoW is a nice campaign but its mp version has flaws of which i think at lest one cannot be fixed:

1) The number of players changes during the campaign, while at first this does't sound like a big problem , i was often at the 1.12 mp server and observed that players _never_ wait for a third player joining when advancing to scenario 4, and continue playing the campaign with a 2 players, baiscally plaing 3-4 player maps with 2 players.

2) In the later part where you have 4 players, people basicalyl play a map that was designed for 1 side as a 4-sides map, resulting in people just contrilling just 3 units per side.

3) It's incomplate, if in the later scenario Landar wins, the campaign just ends , it continues when Kanlenz  wins It shoudl have a branch scenario for the case that landar wins.

4) It's generally unbalanced, there is also already a seperate issue about that.

So i think that we should have a mp campaign that is not a mp version of a sp campaign,

**Comment by CelticMinstrel** [comment]
I agree. In fact, I'd go further and suggest we have two:

* A short, probably about 3-scenario competitive campaign
* A slightly longer coop campaign.

**Comment by richardg999** [comment]
Although the issues mentioned above about LoW have mostly been fixed, I think it makes sense to add an additional mp campaign to mainline.

**Comment by gfgtdf** [comment]
> Although the issues mentioned above about LoW have mostly been fixed

Afaik they haven't

**Comment by richardg999** [comment]
> > Although the issues mentioned above about LoW have mostly been fixed
> 
> Afaik they haven't

I played LoW multiplayer 1 year ago, so I am relying upon memory. What I recall was that it was a 2 player campaign the whole way through (I think one of us just received Cleodil). Also, in the last scenario, the humans were controlling Kalenz and Cleodil, and the AI was controlling Landar, so it was clear that the objective was just to defeat Landar. I also felt that it was fairly balanced the whole way through, but I don't really remember much about the specific scenarios. Is there a resource that you can point me to that describes multiplayer LoW in detail?

**Comment by gfgtdf** [comment]
> What I recall was that it was a 2 player campaign the whole way through (I think one of us just received Cleodil).

hmm from looking the the code this i wrong, espcaiyll in cajhpter 4 it still has playable sides Landar, Kalenz, Olurf and Cleodil

> Also, in the last scenario, the humans were controlling Kalenz and Cleodil, and the AI was controlling Landar

from looking the the code this is partly true: in scenari 21 the players control Kalenz and Cleodil while in scenario 22 and 23 they controll Kalanz and Landar.


> Is there a resource that you can point me to that describes multiplayer LoW in detail?

Dunno i'd just look at the source code https://github.com/wesnoth/wesnoth/tree/master/data/campaigns/Legend_of_Wesmere

**Comment by gfgtdf** [comment]
> What about World Conquest II?

As the maintainer of this add-on i don't have a problem with mainlining it, But i'd probably still like to have an additional story-driven mp campaign in mainline.

**Comment by soliton-** [comment]
Certainly. There is absolutely no reason to stop there. As CelticMinstrel said a competitive campaign would be nice as well.

**Comment by knyghtmare** [comment]
> Is it reworked as a co-op campaign, or competitive?

Co-operative focus.
